tests/config/test_config_parser.py

```diff
@@ -21,3 +22,13 @@ def mock_update(*args, **kwargs):
     monkeypatch.setattr(YAMLConfigParser, "update", mock_update)
```

Comment:
Guess you could use a `lambda` here instead of defining a function that you only use once...

Reply:
Will study about lambdas and then change it :)